roles/poi/files/themes/stadtnavi/topics/pois.lua

```diff
@@ -0,0 +1,261 @@
+        { column = 'name',          type = 'text' }, -- filled if attribute class "name" is set
+        { column = 'website',       type = 'text' }, -- filled if attribute class "contact" is set
+        { column = 'phone',         type = 'text' }, -- filled if attribute class "contact" is set
+        { column = 'email',         type = 'text' }, -- filled if attribute class "contact" is set
+        { column = 'address',       type = 'text' }, -- filled if attribute class "address" is set
+        { column = 'wheelchair',    type = 'text' }, -- filled if attribute class "wheelchair" is set
+        { column = 'opening_hours', type = 'text' }, -- filled if attribute class "opening_hours" is set
+        { column = 'cuisine',       type = 'text' }, -- filled if attribute class "cuisine" is set
+        { column = 'brand',         type = 'text' }, -- filled if attribute class "brand" is set
+        { column = 'operator',      type = 'text' }, -- filled if attribute class "operator" is set
+        { column = 'fee',           type = 'boolean' }, -- filled if attribute class "fee" is set
+    }),
+}
+
+-- taken from https://www.lua.org/pil/20.4.html
+local function from_csv(s)
+    s = s .. ','        -- ending comma
+    local t = {}        -- table to collect fields
+    local fieldstart = 1
+    repeat
+    -- next field is quoted? (start with `"'?)
+    if string.find(s, '^"', fieldstart) then
+        local a, c
+        local i  = fieldstart
+        repeat
+            -- find closing quote
+            a, i, c = string.find(s, '"("?)', i+1)
+        until c ~= '"'    -- quote not followed by quote?
+        if not i then error('unmatched "') end
+        local f = string.sub(s, fieldstart+1, i-1)
+        table.insert(t, (string.gsub(f, '""', '"')))
+        fieldstart = string.find(s, ',', i) + 1
+    else                -- unquoted; find next comma
+        local nexti = string.find(s, ',', fieldstart)
+        table.insert(t, string.sub(s, fieldstart, nexti-1))
+        fieldstart = nexti + 1
+    end
+    until fieldstart > string.len(s)
+    return t
+end
+
+local map_columns = {
+    Oberkategorie = 'category1',
+    Kategorie = 'category2',
+    Unterkategorie = 'category3',
+    ['OSM-Filter'] = 'condition',
+    Eigenschaften = 'attributes',
+}
+
+local function read_config(filename)
+    local config = {}
+    local columns = {}
+
+    for line in io.lines(filename) do
+        local d = from_csv(line)
+        if #columns == 0 then
+            columns = d
+        else
+            local l = {}
+            for idx, c in ipairs(d) do
+                l[map_columns[columns[idx]] or columns[idx]] = c
+            end
+            config[#config + 1] = l
+        end
+    end
+
+    return config
+end
+
+local poi_config = read_config(cfg.config)
+
+-- ---------------------------------------------------------------------------
+
+local map_attributes = {
+    ['küche'] = 'cuisine',
+    ['Öffnungszeiten'] = 'opening_hours',
```

Comment:
All other values are lowercase. Is uppercase ok here? Explicitly required?

Reply:
I have no idea. @joto?

Reply:
Not sure myself. :-) Im am lowercasing the string to be more robust but I don't think this will work with Umlauts. We should get rid of the umlauts, maybe directly use the english names in the config.

Reply:
I switched the CSV file to English attribute names (osm keys).